inspec.yml

```
@@ -441,10 +441,16 @@ inputs:
     value: "aide"
 
   # SV-230484
-  - name: authoritative_timeserver
-    description: Timeserver used in /etc/chrony.conf
-    type: String
-    value: 0.us.pool.ntp.mil
+  - name: authoritative_timeservers
+    description: Timeserver(s) used in /etc/chrony.conf
+    type: Array
+    value: ["0.us.pool.ntp.mil"]
+  
+  # SV-230484
+  - name: authoritative_timeservers_exact
```

```suggestion
  - name: exact_authoritative_timeservers_match_enabled
```

Or `match_all_authoritative_timeservers_exact`
Or `all_authoritative_timeservers_must_match`

These may not be good names, but I would consider changing the name to make it more clear that this is a boolean value. Based on the code, I expected it to be a list of exact authoritative timeservers.